web/classroom_views.py

```
@@ -0,0 +1,101 @@
+                'position_x': data['position']['x'],
+                'position_y': data['position']['y']
+            }
+        )
+        return JsonResponse({'status': 'success'})
+    except Exception as e:
+        return JsonResponse({'status': 'error', 'message': str(e)}, status=400)
+
+
+@login_required
+@require_POST
+def update_chair_state(request):
```

Ruff (0.8.2)

Missing return type annotation for public function `update_chair_state`

(ANN201)